Source/Scene/FrameState.js

```diff
@@ -317,6 +317,13 @@ define([
          */
         this.backgroundColor = undefined;
 
+        /**
+         * The color of the light emitted by the sun.
+         *
+         * @type {Color}
+         */
+        this.sunColor = undefined;
```

Comment:
I wonder if this should tie into https://github.com/AnalyticalGraphicsInc/cesium/pull/7025